src/components/Dashboard/Dashboard.module.scss

```diff
@@ -0,0 +1,57 @@
+@import '../../styles/color.scss';
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+}
+
+.box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 80vw;
+  padding: 1rem;
+  background-color: $spruce-stone;
+  border-radius: 10px;
+}
+
+.title {
+  font-size: 1rem;
+}
+
+.activities {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+  padding: 0;
+  margin: 1.5rem 0;
+  list-style-type: none;
+  cursor: pointer;
```

Comment:
I dont think the whole container should have a pointer cursor. It indicates some interactivity but it is not clickable